eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/MutableMultimap.java

```diff
@@ -77,6 +77,16 @@ default boolean putAllPairs(Iterable<? extends Pair<? extends K, ? extends V>> p
 
     RichIterable<V> removeAll(Object key);
 
+    /**
+     * Get and return a view onto the values associated with the specified key.
+     * Alternatively, if there are no values in the multimap at the key, return an unmodifiable view of specified values,
+     * and put those values in the multimap at the specified key.
+     *
+     * @throws IllegalArgumentException when there are no values for the key and the passed values are empty
+     * @since 10.0
+     */
```

Comment:
I believe the Javadoc needs to be updated to not have a throws clause now. I'd suggest language like:

Puts `values` into multimap if there are no values already associated with `key`. Then returns a view of the values associated with `key`, like the result of get(K).

Reply:
Fixed.